2.4/alpine/Dockerfile

```diff
@@ -27,7 +27,6 @@ RUN set -eux; \
 	runDeps=' \
 		apr \
 		apr-util \
-		apr-util-dbm_db \
```

Comment:
It looks like this was added in #154 for `mod_dav` -- does that work without this now? :see_no_evil: :innocent:

Reply:
The package has been [removed upstream](https://git.alpinelinux.org/aports/commit/main/apr-util?id=5f13bd268c7d86ade44d1bf5c4b2ac3925877626) and Alpine's [apache2-webdav](https://pkgs.alpinelinux.org/package/edge/main/x86_64/apache2-webdav) doesn't depend on it (only on `apr-util`). Might be still worth to double check. Do you know a common testcase?

Reply:
So it likely will work fine since `dbm_db` (aka Berkeley DB <= 5.3) was replaced by `ndbm` in Alpine's build of apr:
```diff
-		--with-dbm=db53 \
-		--with-berkeley-db \
-		--without-gdbm
+		--without-berkeley-db \
+		--with-dbm=ndbm

```

LGTM